### PR TITLE
Series Nav fixes

### PR DESCRIPTION
**Pull request by ovbm**
- remove inverted current episode style and replace with medium label (leaves paynote as is for now)
before:
Screenshot 2021-06-15 at 10 12 18
after:
Screenshot 2021-06-15 at 10 12 28
before:
Screenshot 2021-06-15 at 10 11 48
after:
Screenshot 2021-06-15 at 10 11 29
before:
Screenshot 2021-06-15 at 10 15 41
after:
Screenshot 2021-06-15 at 10 15 34

- adds link to overview to inline description
Screenshot 2021-06-15 at 10 20 00

- adds minWidth to tiles 
before
Screenshot 2021-06-15 at 10 22 26
after
Screenshot 2021-06-15 at 10 22 02

### CODE REVIEW

File: src/components/SeriesNav/SeriesNav.js [hunk 1]
```diff
@@ -111,7 +109,19 @@ function SeriesNav({
...
             </InfoBoxTitle>
-            <InfoBoxText>{series.description}</InfoBoxText>
+            <InfoBoxText>
+              <span>
+                {series.description}
+                {titlePath && (
+                  <>
+                    {' '}
+                    <Link href={titlePath} passHref>
+                      <Editorial.A>Zur Serien-Übersicht</Editorial.A>
```
Comment (tpreusse) [inline src/components/SeriesNav/SeriesNav.js:119]: Is that wording checked with CR/Produktion? We seem to normally write it `Serienübersicht`:
Screenshot 2021-06-15 at 14 43 34
Source: [«Frauenstimmen – Auftakt»](https://www.republik.ch/2021/02/05/auftakt-frauenstimmrecht)

File: src/components/SeriesNav/SeriesNav.js [hunk 2]
```diff
@@ -111,7 +109,19 @@ function SeriesNav({
                 series.title
               )}
             </InfoBoxTitle>
-            <InfoBoxText>{series.description}</InfoBoxText>
+            <InfoBoxText>
+              <span>
```
Comment (tpreusse) [inline src/components/SeriesNav/SeriesNav.js:113]: no span needed as far as i can see

File: src/components/SeriesNav/SeriesNav.js [hunk 3]
```diff
@@ -144,7 +154,7 @@ function SeriesNav({
       </TeaserCarousel>
 
       {inline && PayNote && (
-        <div {...colorScheme.set('backgroundColor', 'defaultInverted')}>
+        <div>
```
Comment (tpreusse) [inline src/components/SeriesNav/SeriesNav.js:157]: you can drop the whole div here

File: src/components/SeriesNav/SeriesNavTileContent.js [hunk 4]
```diff
@@ -81,7 +89,11 @@ const SeriesNavTileContent = ({
   if (inline) {
     return (
       <LinkContainer>
-        <p {...styles.label} {...colorScheme.set('color', 'text')}>
+        <p
+          {...styles.label}
+          {...(current && styles.currentLabel)}
```
Comment (tpreusse) [inline src/components/SeriesNav/SeriesNavTileContent.js:94]: these styles compete for priority regarding font style and `currentLabel` only wins if it's `css` call happens to come after the label css call. Which actually can't be guaranteed (same style leading to the same hash could be used else where and SSR css order is not stable)

Comment (tpreusse) [reply src/components/SeriesNav/SeriesNavTileContent.js:94]: Two possible solutions, isolate font style into separate css calls:
```
  labelBase: css({
    marginTop: 0,
    overflow: 'hidden',
    textOverflow: 'ellipsis',
    whiteSpace: 'nowrap'
  }),
  labelRegular: css({
    ...sansSerifRegular12,
    [mUp]: {
      ...sansSerifRegular14
    }
  }),
  labelMedium: css({
    ...sansSerifMedium12,
    [mUp]: {
      ...sansSerifMedium14
    }
  }),
```

and then
```suggestion
          {...styles.labelbase}
          {...(current ? styles.labelMedium :  styles.labelRegular)}
```


Or use `merge` from glamor:
```suggestion
          {...merge(styles.label, current && styles.currentLabel)}
```
which guarantees the order – let's medium always win.

File: src/components/SeriesNav/SeriesNav.js (hunk 1 above)
Comment (ovbm) [reply src/components/SeriesNav/SeriesNav.js:119]: yes, and forgot to put it in translate as well XD

**Comment by ovbm** [comment]
@tpreusse added translations

File: src/components/SeriesNav/SeriesNavTileContent.js [hunk 5]
```diff
@@ -81,8 +92,13 @@ const SeriesNavTileContent = ({
...
       <LinkContainer>
-        <p {...styles.label} {...colorScheme.set('color', 'text')}>
-          {`${current ? 'Sie lesen: ' : ''}${episode.label}`}
+        <p
+          {...styles.label}
+          {...(current ? styles.labelMedium : styles.labelRegular)}
+          {...colorScheme.set('color', 'text')}
+        >
+          {current ? `${t('styleguide/SeriesNav/current')} ` : null}
+          {episode.label}
```
Comment (tpreusse) [inline src/components/SeriesNav/SeriesNavTileContent.js:101]: `{current ? '' : null}` was causing a annoying server/client html diff warning (react bug), let's use a single string:

```suggestion
          {`${current ? `${t('styleguide/SeriesNav/current')} ` : ''}${episode.label}`}
```

File: src/templates/Article/teasers.js [hunk 6]
```diff
@@ -537,7 +537,8 @@ const createTeasers = ({
         inline: !node.data.grid,
         ActionBar: ActionBar,
         PayNote: PayNote,
-        Link: Link
+        Link: Link,
+        t: t
```
Comment (tpreusse) [inline src/templates/Article/teasers.js:541]: I know this is a black magic shitty problem but we need to do it slightly differently to work in Publikator where props fn is not used. Inject it via the component function without making it part of the `props` function and object:
```
    component: ({ ...props }) => {
      return <SeriesNav t={t} {...props} />
    },
```

**Review by tpreusse — COMMENTED** [review]
Thank you for moving it to the translation file. Two more small tweaks

File: src/templates/Article/teasers.js (hunk 6 above)
Comment (ovbm) [reply src/templates/Article/teasers.js:541]: ok

File: src/components/SeriesNav/SeriesNavTileContent.js (hunk 5 above)
Comment (ovbm) [reply src/components/SeriesNav/SeriesNavTileContent.js]: done

**Review by tpreusse — APPROVED** [review]

**Pull request merged**